config/initializers/active_record_log_subscriber.rb

```diff
@@ -0,0 +1,16 @@
+module LogQuerySource
```

Comment:
@zz9pzza Can you add some comments to this? What are we changing and why are we changing it?